src/components/CustomAutocomplete.tsx

```diff
@@ -51,12 +52,12 @@ export default function AutocompleteFilter(
   );
 }
 
-AutocompleteFilter.propTypes = {
+CustomAutocomplete.propTypes = {
   limitReached: PropTypes.bool,
   placeholder: PropTypes.string,
 };
 
-AutocompleteFilter.defaultProps = {
+CustomAutocomplete.defaultProps = {
   limitReached: false,
   placeholder: undefined,
 };
```

Comment:
Why does `CustomAutoCompete` have propTypes and defaultProps?

Reply:
yeah I guess it doesn't need them.